ui/src/app/components/runtime-configuration-panel.tsx

```diff
@@ -701,36 +699,17 @@ const PanelMain = fp.flow(
                       validMachineTypes={validMainMachineTypes}
                       machineType={analysisConfig.machine.name}
                     />
-                    {enablePersistentDisk || (
```

Comment:
This construction is a little odd.  I *think* what it's doing is: render this element when enablePersistentDisk is false (which is never)